index.html

```diff
@@ -208,91 +285,136 @@ <h2>
+                event sent and the promise must <a>Resolve</a>.
+              </li>
+              <li data-link-for="Document">
+                If the request was not started from an <a>engagement gesture</a> and 
+                the Document has not previously released a successful Pointer Lock with 
+                {{exitPointerLock()}} queue <a>pointerlockerror</a> and 
+                <a href="https://heycam.github.io/webidl/#reject">reject</a> the promise 
+                with a "{{ConstraintError}}" {{DOMException}}.
+                <aside class="note" data-link-for="Document">
+                  <p>
+                    This ensures a user can leave a document that constantly attempts
+                    to lock the pointer. The document will be blocked from locking
+                    upon initial navigation or re-acquiring lock unless the user
+                    re-engages directly with the document.
+                  </p>
+                  <p>
+                    Conversely, if pointer lock is exited via {{exitPointerLock()}}
+                    no <a>engagement gesture</a> is required to reenter pointer lock.
+                    This enables applications that frequently move between
+                    interaction modes, and ones that may do so based on a timer or
+                    remote network activity.
+                  </p>
+                  <p data-link-for="Element">
+                    Pointer lock is commonly combined with fullscreen [[FULLSCREEN]],
+                    and if an <a>engagement gesture</a> is used to enter fullscreen
+                    it is sufficient for a subsequent {{requestPointerLock()}}.
+                  </p>
+                </aside>
+              </li>
+              <li>
+                if the document object's
+                <a data-cite="html/origin.html#active-sandboxing-flag-set">active
+                sandboxing flag set</a> has the <a>sandboxed pointer lock browsing
+                context flag</a> set queue <a>pointerlockerror</a> and 
+                <a href="https://heycam.github.io/webidl/#reject">reject</a> the 
+                promise with a "{{SecurityError}}" {{DOMException}}.
+              </li>
+              <li>
+                <dl data-dfn-for="PointerLockOptions" data-link-for=
+                "PointerLockOptions">
+                  If if <a>unadjustedMovement</a> is true and the platform does not 
+                  support <a>unadjustedMovement</a> queue <a>pointerlockerror</a> and 
+                  <a href="https://heycam.github.io/webidl/#reject">reject</a> the 
+                  promise with a "{{NotSupportedError}}" {{DOMException}}.
+                </dl>
+                <aside class="note" data-link-for="Document">
+                  "{{NotSupportedError}}" {{DOMException}} should be reserved for 
+                  unsupported options in future iterations of this spec.
```

Comment:
I'd be a bit more explicit, e.g. "That is, no other error conditions should use "NotSupportedError", so that web developers can use it for feature testing"